hnn_core/gui/gui.py

```diff
@@ -674,15 +674,15 @@ def _get_connectivity_widgets(conn_data):
                                  description="weight",
                                  style=style)
 
-        w_slider = FloatLogSlider(value=conn_data[receptor_name]['weight'],
-                                  min=-5, max=1, step=0.2,
-                                  description=" ",
-                                  disabled=False,
-                                  continuous_update=False,
-                                  orientation='horizontal',
-                                  readout=False,
-                                  readout_format='.2e',
-                                  style=style)
+        w_slider = FloatSlider(value=conn_data[receptor_name]['weight'],
+                               min=-1, max=1, step=0.01,
```

Comment:
not sure if this is the right fix. This will prevent user from making finer adjustments.

Also I guess weights can't be negative.

Wondering if the slider and the stepper should have different step sizes ... large logarithmic step size for the slider and small step size for the stepper arrows for finer adjustment?

any opinion @rythorpe @ntolley ?

Reply:
^This sounds like a fair suggestion to me. This will be consistent across local network and drive connectivity right?

Reply:
It's not negative values that break the thing. Sometimes we have zero values. With `FloatLogSlider` they are all strictly >0. That actually will cause problems in some settings.

Reply:
as we talked in the meeting today, let's just get rid of the sliders.